portal-api/api-tests/v1/users/reset-password.api-test.js

```diff
@@ -58,80 +68,88 @@ describe('password reset', () => {
   });
 
   it('should return the correct user for a given reset token', async () => {
-    const reset = await resetPassword(MOCK_USER.email);
-    const user = await getUserByPasswordToken(reset.resetToken);
+    const passwordResetToken = 'passwordResetToken';
+    utils.genPasswordResetToken = mockKnownTokenResponse(passwordResetToken);
```

Comment:
I love the change here to not have to return the reset password token just for the tests anymore
Is `utils.genPasswordResetToken` being restored to its actual implementation after the tests somehow? I couldn't spot how if so